packages/pds/src/db/migrations/20230613T164932261Z-init.ts

```diff
@@ -0,0 +1,375 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+// @TODO make takedownId a varchar w/o fkey?
+
+export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
+  const binaryDatatype = dialect === 'sqlite' ? 'blob' : sql`bytea`
+
+  await db.schema
+    .createTable('app_migration')
+    .addColumn('id', 'varchar', (col) => col.primaryKey())
+    .addColumn('success', 'int2', (col) => col.notNull().defaultTo(0))
+    .addColumn('completedAt', 'varchar', (col) => col)
+    .execute()
```

Comment:
I wonder if we can get rid of this table and the app migration system.

Reply:
maybe? it might be helpful with some automatic migrations for the pds distribution in the future tho 🤔 

I think the runtime flag table is asking for it more tbh